google/cloud/bigtable/row_data.py

```diff
@@ -507,7 +508,16 @@ def _read_next(self):
         try:
             return six.next(self.response_iterator)
         except grpc.RpcError as grpc_error:
-            raise exceptions.from_grpc_error(grpc_error)
+            # TODO: this needs to be moved to a more general location (ie interceptor)
+            e = exceptions.from_grpc_error(grpc_error)
+            # Sometimes GOAWAYs are surfaced as INTERNAL errors, which makes
+            # them unretriable. This patches that behavior
+            if (
+                e.grpc_status_code == grpc.StatusCode.INTERNAL
+                and "RST_STREAM" in e.message
```

Comment:
I think the error messages comes in different formats like "Rst stream" "RST_STREAM" etc. maybe do: e.message.tolowercase and check for both "rst stream" and "rst_stream".

Reply:
done